### ISSUE TITLE
[timm] `crossvit_9_240` fails accuracy

### ISSUE THREAD

**Issue by whitneywhtsang** [intel/intel-xpu-backend-for-triton#527]
`crossvit_9_240` fails accuracy in all modes and data types.

**Comment by ienkovich** [comment]
I ran this test with `TIMM_FUSED_ATTN=0` due to the previously found problem in TIMM benchmarks. With this option CPU results match in eager inductor modes. XPU results differ but XPU inductor results match CPU results, so I assume the problem is in XPU eager mode. Used reproducer:
```
import copy
import torch
import intel_extension_for_pytorch
import timm
from timm.models import create_model

from torch._dynamo.utils import clone_inputs
from torch._dynamo.testing import same, reset_rng_state

model_name = "crossvit_9_240"

def make_model():
    reset_rng_state()
    return create_model(model_name, in_chans=3, scriptable=False, num_classes=None, drop_rate=0.0,
            drop_path_rate=None,
            drop_block_rate=None,
            pretrained=False)

def make_input(device):
    input_size=(3, 240, 240)
    batch_size = 8

    # generate input data
    torch.manual_seed(1337)
    input_tensor = torch.randint(
        256, size=(batch_size,) + input_size, device='cpu'
    ).to(dtype=torch.float32).to(device)
    mean = torch.mean(input_tensor)
    std_dev = torch.std(input_tensor)
    res = (input_tensor - mean) / std_dev
    return [res]

def run_model(model, device, inputs):
    model_copy = copy.deepcopy(model).to(device)
    model.eval()
    input_copy = clone_inputs(inputs)
    with torch.no_grad():
        return model_copy(*input_copy)

def check_model(model, device):
    input = make_input(device)

    reset_rng_state()

    res1 = run_model(model, device, input)

    torch._dynamo.reset()
    reset_rng_state()

    optimize_ctx = torch._dynamo.optimize('inductor', nopython=False)
    model_fn = optimize_ctx(run_model)
    res2 = model_fn(model, device, input)

    print(f"{device} results match: {same(res1, res2)}")

    return res1, res2

model = make_model()

cpu1, cpu2 = check_model(model, 'cpu')
xpu1, xpu2 = check_model(model, 'xpu')

print(f"Eager results match: {same(cpu1, xpu1.to('cpu'))}")
print(f"Inductor results match: {same(cpu2, xpu2.to('cpu'))}")
```
I could reduce the model to a single interpolation layer with the same situation but other problems might also exist. Note that batch size >1 is required to reproduce the problem because the first result in a batch is correct. Small reproducer:
```
import torch
import intel_extension_for_pytorch

class Test(torch.nn.Module):
    def __init__(self):
        super().__init__()

    def forward(self, x):
        x = torch.nn.functional.interpolate(x, size=(224, 224), mode='bicubic', align_corners=False)
        return x

def make_input(device):
    torch.manual_seed(1337)
    return torch.randn((2, 3, 240, 240), dtype=torch.float32, device='cpu').to(device)

def check_model(device):
    model = Test().to(device)
    input = make_input(device)

    res1 = model(input)
    
    model = torch.compile(model, backend='inductor')
    res2 = model(input)

    print(f"{device} results match: {torch.allclose(res1, res2, atol=1e-4, rtol=1e-4)}")

    return res1, res2

cpu1, cpu2 = check_model('cpu')
xpu1, xpu2 = check_model('xpu')

print(f"Eager results match: {torch.allclose(cpu1, xpu1.to('cpu'), atol=1e-4, rtol=1e-4)}")
print(f"Inductor results match: {torch.allclose(cpu2, xpu2.to('cpu'), atol=1e-4, rtol=1e-4)}")

```

**Comment by vlad-penkin** [comment]
Issue no longer reproducible

Env:
- pytorch is built from source, top of the main trunk, commit_id - `9a8ab778d34bd24c5caceb340837483decc4c311`
- triton xpu is built from source, top of the main trunk, commit_id - `fe93a00ffe438e9ba8c8392c0b051b1662c810de`
- benchmark is built from source, top of the main trunk, commit_id - `d54ca9f80ead108c8797441681e219becaf963d8`
- torchaudio is built from source, top of the main trunk,  commit_id - `1980f8af5bcd0bb2ce51965cf79d8d4c25dad8a0`
- torchvision  is built from source, top of the main trunk,  commit_id - `10239873229e527f8b7e7b3340c40ee38bb1cfc4`
- PyTorch Dependency Bundle 0.5.0
- Latest Rolling Driver
